Apps/W1/Shopify/app/src/Companies/Interfaces/ShpfyICustomerCompanyMapping.Interface.al

```diff
@@ -0,0 +1,11 @@
+namespace Microsoft.Integration.Shopify;
+
+/// <summary>
+/// Interface "Shpfy ICustomer/Company Mapping."
+/// </summary>
+interface "Shpfy ICustomer/Company Mapping"
```

Comment:
Not quite what I had in mind. You don't want to replace the interface, you want to add to it. So all the other implementations just have to add the new procedure.
So the new interface would only implement the Find mapping.

Should we maybe have a quick call about this? I'm again ping-ponging in my head if we should go for this approach, or the interface extension one...
Both would work equally well, though MS has made the extensions for a reason...